src/main/webapp/style.css

```diff
@@ -487,12 +668,43 @@ Main Content Container Styling
       font-size: 16pt;
     }
 
-    .zipCode-submit-btn {
+    .search-button {
       font-size: 16pt;
     }
 
     .results-card {
       column-gap: 15px;
       row-gap: 15px;
     }
+
+    #popup-form-container {
+      height: 40%;
+      width: 15%;
+    }
+}
+
+@media screen 
+  and (max-device-height: 900px) {
```

Comment:
nit: I think we talked about preferring max-[height|width] here, and below.

Reply:
Fixed